test/server/cards/02_SHD/units/KintanIntimidator.spec.ts

```diff
@@ -0,0 +1,31 @@
+describe('Kintan Intimidator', function () {
+    integration(function (contextRef) {
+        describe('Kintan Intimidator\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['kintan-intimidator']
+                    },
+                    player2: {
+                        groundArena: ['battlefield-marine']
+                    }
+                });
+            });
+
+            it('should exhaust the defender when attacking a unit', function () {
+                const { context } = contextRef;
+
+                // Select Kintan Intimidator
+                expect(context.player1).toBeAbleToSelectExactly([context.kintanIntimidator]);
```

Comment:
We should only use `expect` if we're testing the card ability. This check and the check on line 22 are validating basic stuff like attack that we don't need to check here, it's checked in other places.